scripts/gitlab/push.sh

```diff
@@ -2,11 +2,17 @@
 
 set -e # fail on any error
 set -u # treat unset variables as error
-updater_push_release () {
-  echo "push release"
-  # Mainnet
 
-}
+
+echo "__________Register Release__________"
+DATA="secret=$RELEASES_SECRET"
+
+echo "Pushing release to Mainnet"
+../../scripts/gitlab/safe_curl.sh $DATA "http://update.parity.io:1337/push-release/$CI_BUILD_REF_NAME/$CI_BUILD_REF"
```

Comment:
Thank you for the fix! Just the path should be `./scripts/gitlab/safe_curl.sh` here (same for the line 14)

Reply:
wish we had someone at parity who knows bash :man_facepalming: k